src/main.rs

```diff
@@ -54,9 +54,9 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     let (sender, receiver) = channel();
 
-    let mut tmux_daemon_attached = TmuxDaemon::new(&tmux_context.session)?;
+    let mut tmux_daemon_attached = TmuxDaemon::new(&tmux_context.session_id)?;
     tmux_daemon_attached.listen_for_dead_panes(sender.clone())?;
-    let mut tmux_daemon_detached = TmuxDaemon::new(&tmux_context.detached_session)?;
+    let mut tmux_daemon_detached = TmuxDaemon::new(&tmux_context.detached_session_id)?;
```

Comment:
Have a weird bug here, whichever session I try to create a subscription to listen for pane dead notifications for first will fail, with the following error:

```
not a control client
```

In this order, it fails for the attached session, if I reverse the order it will fail for the detached session.

Reply:
I think this might have something to do with my environment. I checked out earlier commits that I know were working previously, they no longer do.

Reply:
interesting ill keep an eye out for that, I havent seen it so far though